<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DIGAMBAR-PC" ignored="0" name="Login Test" tests="6" failures="1" timestamp="17 Nov 2024 10:58:14 GMT" time="246.89" errors="0">
  <testcase name="loginAndForgotPassword" time="74.465" classname="tests.LoginTest">
    <skipped/>
  </testcase> <!-- loginAndForgotPassword -->
  <testcase name="loginAndForgotPassword" time="0.016" classname="tests.LoginTest">
    <skipped/>
  </testcase> <!-- loginAndForgotPassword -->
  <testcase name="loginAndForgotPassword" time="0.017" classname="tests.LoginTest">
    <skipped/>
  </testcase> <!-- loginAndForgotPassword -->
  <testcase name="loginAndForgotPassword" time="0.015" classname="tests.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.LoginTest.loginAndForgotPassword(LoginTest.java:24)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginAndForgotPassword -->
  <testcase name="method2" time="0.002" classname="tests.LoginTest"/>
  <testcase name="method3" time="0.001" classname="tests.LoginTest"/>
</testsuite> <!-- Login Test -->
